lua/cinema/theater/services/sh_bflix.lua

```diff
@@ -119,28 +128,19 @@ if CLIENT then
     end
 
     function SERVICE:LoadVideo(Video, panel)
-        panel:EnsureURL(Video:Key())
+        panel:EnsureURL(Video:Data())
```

Comment:
You should be checking if Data ~= "" here. Data is just an empty string, not the Key, by default

Reply:
done